flink-runtime/src/main/java/org/apache/flink/runtime/clusterframework/overlays/FlinkDistributionOverlay.java

```diff
@@ -91,7 +96,6 @@ public static Builder newBuilder() {
 		File flinkLibPath;
 		@Nullable
 		File flinkPluginsPath;
-
```

Comment:
This change isn't needed. Imo, we should revert it.

Reply:
I got it.